record-and-playback/screenshare/scripts/publish/screenshare.rb

```
@@ -74,7 +74,7 @@
 end
 
 # Captions files
-captions = JSON.load(File.new("#{process_dir}/captions.json", 'r'))
+captions = JSON.load(File.read("#{process_dir}/captions.json"))
```

RuboCop (1.86.2)

[convention] Prefer `JSON.parse` over `JSON.load`.

(Security/JSONLoad)